src/main/java/io/github/arrayv/prompts/SortPrompt.java

```diff
@@ -24,22 +26,11 @@
 import io.github.arrayv.dialogs.ImportSortDialog;
 import io.github.arrayv.frames.AppFrame;
 import io.github.arrayv.frames.UtilFrame;
+import io.github.arrayv.groovyapi.GroovyLocals;
 import io.github.arrayv.main.ArrayVisualizer;
 import io.github.arrayv.main.SortAnalyzer;
 import io.github.arrayv.panes.JErrorPane;
 import io.github.arrayv.sortdata.SortInfo;
-import io.github.arrayv.threads.MultipleSortThread;
```

Comment:
shouldn't `MultipleSortThread.java` be deleted, as it does nothing now?

Reply:
It has been removed. However, this leaves `RunSort` by itself in a package. I feel like `RunSort` should be moved somewhere else. Thoughts?

Reply:
maybe `utils`, or `main`? you probably know the structure better though